matrix.yaml

```diff
@@ -65,4 +65,4 @@ exclude:
 
   # exclude ci for ubuntu18.04
```

Comment:
can you update the comments in this file that reference `ci` to `ci-conda`?

it looks like there are a few